internal/exec/stages/files/files.go

```diff
@@ -60,15 +60,23 @@ func (creator) Name() string {
 type stage struct {
 	util.Util
 	toRelabel []string
+	isLive    bool
 }
 
 func (stage) Name() string {
 	return name
 }
 
+func (s stage) ApplyLive(config types.Config) error {
+	s.isLive = true
+	return s.Run(config)
+}
+
 func (s stage) Run(config types.Config) error {
-	if err := s.checkRelabeling(); err != nil {
-		return fmt.Errorf("failed to check if SELinux labeling required: %v", err)
+	if !s.isLive {
+		if err := s.checkRelabeling(); err != nil {
+			return fmt.Errorf("failed to check if SELinux labeling required: %v", err)
+		}
```

Comment:
Right now, ostree will perform labeling at client time, but in the future for derived layers https://github.com/ostreedev/ostree-rs-ext/issues/159 will push it server side.

But this is tricky since if e.g. doing a live-applied ignition to a booted physical/VM system, we actually do want ignition to perform labeling right?  (Since there's no ostree/rpm involved).

Probably the more correct thing here is to check `selinuxenabled` i.e. the presence of `/sys/fs/selinux`.  That way we'll skip labeling if applied in a container build.

Reply:
> But this is tricky since if e.g. doing a live-applied ignition to a booted physical/VM system, we actually do want ignition to perform labeling right? (Since there's no ostree/rpm involved).

On a fully booted system, SELinux is loaded and initialized, so it should be setting labels based on the policy, right? But yeah, I mostly share @bgilbert's concerns around whether we want to support end-users directly applying this anyway.